builtin/credential/aws/backend_test.go

```diff
@@ -1440,8 +1446,8 @@ func TestBackend_pathStsConfig(t *testing.T) {
 
 	stsReq.Operation = logical.CreateOperation
 	stsReq.Path = "config/sts/account2"
-	stsReq.Data = data
-	// create another entry to test the list operation
+	stsReq.Data = data2
+	// create another entry with alternate data to test ExternalID and LIST
```

Comment:
If possible, could we add a block for checking `"external_id": "fake_id"` existence in a read response before line 1455?

Reply:
added a response check for data2

Reply:
Thank you!